src/webgpu/util/f32_interval.ts

```diff
@@ -13,15 +19,20 @@ export class F32Interval {
    *
    * @param bounds a pair of numbers indicating the beginning then the end of the interval
    */
-  public constructor(...bounds: [number, number]) {
-    const [begin, end] = bounds;
+  public constructor(...bounds: [number] | [number, number]) {
```

Comment:
Allows passing the result of .bounds() into the constructor. I don't think I directly use that anywhere, but it feels like that should have symetry, and probably can be used to clean up other code, like toF32Interval.